__tests__/app.test.js

```diff
@@ -193,6 +193,72 @@ describe("/api/articles/:article_id/comments", () => {
             });
         });
     });
+    describe('POST', ()  => {
+        it('should respond with 201 and add comment for specified article and respond with the posted comment', ()    => {
+            return request(app)
+            .post("/api/articles/7/comments")
+            .send({
+                   username: "lurker",
+                   body: "Hellllloooooo world!"
+                  })
+            .expect(201)
+            .then((response) => {
+                const comment = response.body.addedComment;
+                const convertTime =  convertTimestampToDate(comment.created_at);
+                const date = (Object.values(convertTime).join(''))
+                const currentTime = new Date()
+                expect(Object.keys(comment).length).toBe(6);
+                expect(comment).toMatchObject({
+                    article_id: 7,
+                    author: "lurker",
+                    body: "Hellllloooooo world!",
+                    comment_id: 19,
+                    created_at: date,
+                    votes: 0  
+                });
+        });
+        });
+        it('status code 400 with appropriate message when key is missing from input body', ()    =>   {
+        return request(app)
+        .post("/api/articles/7/comments")
+        .send({
+                body: "bodyodyodyody"
+             })
+        .expect(400)
+        .then((response) =>  {
+                expect(response.body.msg).toBe('Bad request')
+            });
+        });
+        it('status code 404 when invalid value inputted', ()    =>   {
```

Comment:
Could we be more descriptive here? I think what we're testing for is username not found on the users table?